src/Evolution/Systems/GeneralizedHarmonic/TimeDerivative.cpp

```diff
@@ -29,6 +29,36 @@ void TimeDerivative<Dim>::apply(
     const gsl::not_null<tnsr::aa<DataVector, Dim>*> dt_spacetime_metric,
     const gsl::not_null<tnsr::aa<DataVector, Dim>*> dt_pi,
     const gsl::not_null<tnsr::iaa<DataVector, Dim>*> dt_phi,
+    const gsl::not_null<Scalar<DataVector>*> temp_gamma1,
```

Comment:
If functions with this many temporaries will be common, we might want to try to find a way  to pass them in without quite so many function arguments. Not for this PR, just a thought that occurs when there's a function with ~30 args :P.

Reply:
I agree. My current plan was to at some point enable non-owning Variables. That would basically solve this issue while still making everything cache friendly and easy to project to mortars. It will also prove useful for doing things like ADER where you could allocate one gigantic block of memory and then point into it.